app/src/org/commcare/android/database/app/DatabaseAppOpenHelper.java

```diff
@@ -108,6 +108,13 @@ public static String indexOnTableWithPGUIDCommand(String indexName,
         return indexOnTableCommand(indexName, tableName, Resource.META_INDEX_PARENT_GUID);
     }
 
+    /**
+     * Build and return SQL command to add a column to a table
+     */
+    public static String addColumnToTable(String tableName, String columnName, String dataType) {
```

Comment:
I'd move this method to the dbutits, and I think the column name possibly needs to get sanitized, will check usage in code.